src/app/Fake.Core.String/String.fs

```diff
@@ -146,6 +146,9 @@ let convertTextToWindowsLineBreaks text =
     |> replace MacLineBreaks LinuxLineBreaks
     |> replace LinuxLineBreaks WindowsLineBreaks
 
+let splitLines (text:string)=let variants=[|LinuxLineBreaks; WindowsLineBreaks; MacLineBreaks|]
+                             text.Split(variants, StringSplitOptions.RemoveEmptyEntries) |> Array.toList
```

Comment:
Please make the new public API return a sequence instead of a list and remove the toList call?
Also, we should remove the strange formatting (start a new line) and add a documentation string.

Additionally, I don't think we should have "RemoveEmptyEntries" in this API.

Reply:
I've moved the implementation to be local to the PR, instead of adding an additional API method.